ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po

```diff
@@ -248,31 +248,86 @@ msgstr "Supprimer"
 msgid "You haven't created any API Tokens."
 msgstr "Vous n’avez créé aucun jeton d’API."
 
-#: ckanext/canada/strings.py:43
+#: ckanext/canada/strings.py:44
 msgid "Beta"
 msgstr "Bêta"
 
+#: ckanext/canada/strings.py:45
+#, TODO: add translation, python-format
+msgid "Deleted API token %s"
+msgstr ""
```

Comment:
when we have empty values here does it remove translations that were defined in ckan?

Reply:
Not sure, but this string is not in CKAN (yet, there is an upstream PR for that https://github.com/ckan/ckan/pull/7996)

Reply:
ok, if the strings we've added aren't upstream then this is fine once we have translations here.